src/Grid/topology.jl

```diff
@@ -70,9 +70,22 @@ mutable struct ExclusiveTopology <: AbstractTopology
     edge_edge_neighbor::Matrix{EntityNeighborhood{EdgeIndex}}
     # lazy constructed face topology
     face_skeleton::Union{Vector{FaceIndex}, Nothing}
+    edge_skeleton::Union{Vector{EdgeIndex}, Nothing}
+    vertex_skeleton::Union{Vector{VertexIndex}, Nothing}
```

Comment:
I don't think a vertex skeleton is useful. It's jused the unique nodes - higher order nodes, but I guess that's the trade off for one dimension if one wants to do an integral over the boundary between two elements

Reply:
Yes, it is used by e.g. the `InterfaceIterator` in 1d.